buzzmobile/tests/sense/test_inputer.py

```diff
@@ -0,0 +1,55 @@
+import rostest
+import unittest
+import rospy
+import time
+from std_msgs.msg import String
+import roslaunch
+
+PKG = 'buzzmobile'
+NAME = 'test_inputer'
```

Comment:
Isn't there a better way of getting this info than hardcoding it? Does ROS not provide it somehow?